spec/htmldiff_spec.rb

```diff
@@ -46,6 +46,13 @@ class TestDiff
     expect(diff).to eq("a <a href=\"#c2\"></a> <del class=\"diffmod\">b</del><ins class=\"diffmod\">c</ins>")
   end
 
+  it "there should be a bug..." do
```

Comment:
I would like the naming to tell what the bug is.  (if I understand it correctly): That the HTML is illegal (is this correct?)

Reply:
Also, perhaps ad a test and that refutes the behaviour that you consider not buggy